src/main/java/uk/gov/hmcts/befta/player/DefaultBackEndFunctionalTestScenarioPlayer.java

```diff
@@ -75,7 +75,7 @@ public void initializeAppropriateTestContextAsDetailedInTheTestDataSource() {
     }
 
     @Override
-    @Given("a case that has just been created as in [{}]")
```

Comment:
instead of changing the sentence to this kind of a very specific one, we should fix the implementation of the original sentence, if needed.